.github/workflows/publish.yml

```diff
@@ -115,6 +115,11 @@ jobs:
           command: "V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push eve"
           dockerhub-token: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
           dockerhub-account: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
+      - uses: ./.github/actions/run-make
```

Comment:
We did not do this for riscv in the code which we lost. It was
      - name: Build and push collected_sources to eve-source repo
        if: matrix.arch != 'riscv64'
        run: |
          make -e V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push sbom collected_sources compare_sbom_collected_sources publish_sources

I have no idea whether it can even build the sbom and sources for riscv so let's not add that.